common/lib/xmodule/xmodule/js/spec/capa/display_spec.js

```diff
@@ -365,13 +365,19 @@ data-url='/problem/quiz/'> \
         };
         return promise;
       });
-      // Make sure the submit button is enabled before submitting
-      $('#input_example_1').val('test').trigger('input');
-      expect(this.problem.submitButton).not.toHaveAttr('disabled');
-      this.problem.submit();
-      // After submit, the button should not be disabled and should have text as 'Submit'
-      expect(this.problem.submitButtonLabel.text()).toBe('Submit');
-      expect(this.problem.submitButton).not.toHaveAttr('disabled');
+
+     ['Submit', 'Resubmit'].forEach(function(button_label) {
```

Comment:
`button_label` is only being used on line 375. Maybe doing a second submit and checking that the label still remains "Resubmit" would be better?